### ISSUE TITLE
[CORE][SPELL] Blink

### ISSUE THREAD

**Issue by franklampardahar** [azerothcore/azerothcore-wotlk#381]
**Description**: Mage blink Spell

**Current behaviour**: when use blink spell of mage in a slope place not working or getting on the texture. 

**Expected behaviour**: Must work in slope and not go to texture.

**Steps to reproduce the problem**: 

1. learn blink spell go on slope place and use 

**Branch(es)**: master

**AC hash/commit**:  latest AC

**Operating system**:  se7en 64 bit

**Modules**:  Are you using modules? no

**Other Customizations**: Are you using any core/db patches or other customizations? If yes please specify them here.

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing c++ create a Pull Request)

**Comment by DarkmoonRabbit** [comment]
This is a complicated question.

**Comment by ShinDarth** [comment]
@kvipka

**Comment by franklampardahar** [comment]
fixed by trinitycore closed!

**Comment by ShinDarth** [comment]
@franklampardahar an issue should be closed only once the problem has been resolved and the fix merged into the master branch. Since you already tested that fix, can you please [open a PR](https://github.com/azerothcore/azerothcore-wotlk/wiki/Contribute#how-to-create-a-pull-request) so we can solve it?

**Comment by ShinDarth** [comment]
I don't understand. Can you please open a PR?

**Comment by franklampardahar** [comment]
@ShinDarth . Really i don't know how open PR :| first need gitbash?

**Comment by ShinDarth** [comment]
@franklampardahar it's easy and there are many tutorials on the web, for example:

https://help.github.com/articles/creating-a-pull-request-from-a-fork/

let me know if you need further help ;)

**Comment by kvipka** [comment]
wow... if you want - i will prepare PR for you

**Comment by ShinDarth** [comment]
thanks @kvipka

**Comment by franklampardahar** [comment]
@ShinDarth  @kvipka yes can you open PR for me?

**Comment by ShinDarth** [comment]
he did it already: https://github.com/azerothcore/azerothcore-wotlk/pull/404

**Comment by franklampardahar** [comment]
Fixed by @kvipka https://github.com/azerothcore/azerothcore-wotlk/pull/404/commits/fea7cc7775e4bce5aa9a6e304a0b18bad381916a
